examples/go/.envrc

```diff
@@ -0,0 +1,3 @@
+source_url "https://raw.githubusercontent.com/cachix/devenv/95f329d49a8a5289d31e0982652f7058a189bfca/direnvrc" "sha256-d+8cBpDfDBj41inrADaJt+bDWhOktwslgoP5YiGJ1v0="
+
```

Comment:
not sure if we need to remove this file

Reply:
Yeah it's best to leave it out for examples, same for gitignore

Reply:
updated commit without this files.